src/templates/admin/review/projected_issue.html

```diff
@@ -28,13 +28,13 @@ <h2>Help</h2>
 				</div>
 				<div class="content">
 					<p><strong>{% trans "What is this for?" %}</strong></p>
-					<p>{% trans "The projected issue setting helps you track which issue an article is intended to be published with."%}</p>
+					<p>{% trans "The projected issue field is for journals that add articles to issues on a rolling basis and publish them individually, rather than publishing them all at once. It helps you track internally which issue an article is intended to be published with, without displaying an unpublished article under a published issue."%}</p>
```

Comment:
It might be worth commenting that non-Issue based journals can use it to inform typesetters of the issue an article is going to be part of?

Reply:
I'm not sure what you mean by a non-issue-based journal that puts articles into issues. Does rolling publication in chronologically not cover the use case for this field?

Reply:
Or are you saying, just mention communication with typesetters?

Reply:
Me neither. I _think_ it should be that Issue based journals _can_ also use the field...

Reply:
I've reworded it more openly